hack/local-up-cluster.sh

```diff
@@ -609,7 +611,9 @@ EOF
 fi
 }
 
-test_docker
+if [[ "${CONTAINER_RUNTIME}" == "docker" ]] || [[ -z "${CONTAINER_RUNTIME}" ]]; then
+  test_docker
```

Comment:
good question. :) I don't know the answer. 

minor nit: the `[[ -z "${CONTAINER_RUNTIME}" ]]` branch of this probably won't ever apply and can be removed, since we set `CONTAINER_RUNTIME` to `docker` by default if it's empty.